test/e2e_node/image_list.go

```diff
@@ -89,16 +90,12 @@ func updateImageAllowList(ctx context.Context) {
 	} else {
 		e2epod.ImagePrePullList.Insert(gpuDevicePluginImage)
 	}
+	// TODO: unify the sample device plugin image, use the latest for everything
```

Comment:
What prevents us from doing it now?

Reply:
Insufficient testing yet. That's my only remaining concern

Reply:
Would it make sense to create an issue about this?

Reply:
otherwise the PR looks good to me
/lgtm

Reply:
> Would it make sense to create an issue about this?

yup! done: https://github.com/kubernetes/kubernetes/issues/125227